http/data-url.json

```diff
@@ -22,12 +22,24 @@
             "version_added": true,
             "notes": "The maximum size supported is 4GB"
           },
-          "firefox": {
-            "version_added": true
-          },
-          "firefox_android": {
-            "version_added": true
-          },
+          "firefox": [
+            {
+              "version_added": "59",
+              "notes": "Top-level navigation to <code>data://</code> URIs has been blocked (see <a href='https://www.fxsitecompat.com/en-CA/docs/2017/data-url-navigations-on-top-level-window-will-now-be-blocked/'>Data URL navigations on top level window will now be blocked</a> for more details)."
```

Comment:
I would rather link to this: https://blog.mozilla.org/security/2017/11/27/blocking-top-level-navigations-data-urls-firefox-58/